services/wallet/common/log_parser.go

```diff
@@ -49,10 +53,13 @@ const (
 
 	// Transfer (index_topic_1 address from, index_topic_2 address to, uint256 value)
 	// Transfer (index_topic_1 address from, index_topic_2 address to, index_topic_3 uint256 tokenId)
-	Erc20_721TransferEventSignature = "Transfer(address,address,uint256)"
+	Erc20_721TransferEventSignature     = "Transfer(address,address,uint256)"
+	Erc1155TransferSingleEventSignature = "TransferSingle(address,address,address,uint256,uint256)"
```

Comment:
Please add a comment explaining the format of these events, like the signature types have